code/datums/mind.dm

```diff
@@ -132,11 +132,17 @@
 	var/output = "<B>[current.real_name]'s Memories:</B><HR>"
 	output += memory
 
-	if(objectives.len)
-		output += "<HR><B>Objectives:</B><BR>"
+	var/antag_datum_objectives = FALSE
+	for(var/datum/antagonist/A in antag_datums)
+		output += A.antag_memory
+		if(!antag_datum_objectives && LAZYLEN(A.objectives))
+			antag_datum_objectives = TRUE
+
+	if(LAZYLEN(objectives) || antag_datum_objectives)
+		output += "<HR><B>Objectives:</B>"

```

Comment:
Just a minor nitpick, discovered this while testing. We need a line break (\<br>) at the end of this line, before the end quote. Otherwise, people's objective text in their notes will start like this: `Objectives:Objective #1: Description`.